pages/ai-ecosystem/integrations.mdx

```diff
@@ -533,4 +535,226 @@ answer.
 
+
+## Cognee
+
+<Cards>
+  <Cards.Card
+    icon={<GitHub />}
+    title="Cognee integration"
+    href="https://github.com/topoteretes/cognee/tree/main/cognee/infrastructure/databases/graph/memgraph"
+  />
+</Cards>
+
+Cognee is an AI-powered toolkit for cognitive search and graph-based knowledge
+representation. It leverages LLMs to extract structured concepts and
+relationships from natural language, storing them in Memgraph as a knowledge
+graph. This enables semantic search and graph querying over unstructured text.
+You can follow the [Jupyter Notebook
+example](https://github.com/memgraph/ai-demos/blob/main/integrations/cognee/cognee.ipynb)
+or go through quick start below.
+
+{<h3 className="custom-header">Installation</h3>}
+
+Before you begin, ensure the following are installed:
+- **Docker**: [Install Docker](https://docs.docker.com/engine/install/)
+- **Memgraph**: Run Memgraph using the recommended script:
+
+```bash
+# macOS/Linux
+curl https://install.memgraph.com | sh
+
+# Windows
+iwr https://windows.memgraph.com | iex
+```
+
+This launches Memgraph on `localhost:3000`.
+
+- **Poetry and dependencies**:
+```bash
+poetry init
+poetry add cognee
+pip install neo4j
+```
+
+{<h3 className="custom-header">Environment setup</h3>}
+
+Create a `.env` file in your project root and define your configuration:
+
+```env
+LLM_API_KEY=sk-...
+LLM_MODEL=openai/gpt-4o-mini
+LLM_PROVIDER=openai
+EMBEDDING_PROVIDER=openai
+EMBEDDING_MODEL=openai/text-embedding-3-large
+GRAPH_DATABASE_PROVIDER=memgraph
+GRAPH_DATABASE_URL=bolt://localhost:7687
+GRAPH_DATABASE_USERNAME=" "
+GRAPH_DATABASE_PASSWORD=" "
+```
+
+In your script or notebook, load the environment:
+
+```python
+from dotenv import load_dotenv
+load_dotenv()
+```
+
+{<h3 className="custom-header">Using Cognee</h3>}
+
+Once the environment is set up, you can start processing text with Cognee:
+
+```python
+import cognee
+import asyncio
+
+async def run():
+    text = "Natural language processing (NLP) is a subfield of computer science focused on the interaction between computers and human language."
+
+    # Add text to Cognee
+    await cognee.add(text)
+
+    # Generate the knowledge graph with LLMs
+    await cognee.cognify()
+
+    # Semantic search
+    results = await cognee.search(query_text="What is NLP?")
+    
+    print("Search Results:\n")
+    for result in results:
+        print(result)
+
+await run()
+```
+
+{<h4 className="custom-header">What happens under the hood?</h4>}
+
+- `cognee.add()`: Stores unstructured input text.
+- `cognee.cognify()`: Converts text to a structured graph using LLMs and stores
+  it in Memgraph.
+- `cognee.search()`: Runs semantic search queries using vector embeddings and
+  graph context.
+
+
+## Mem0
+
+<Cards>
+  <Cards.Card
+    icon={<GitHub />}
+    title="Mem0 integration"
+    href="https://github.com/mem0ai/mem0/blob/main/mem0/memory/memgraph_memory.py"
+  />
+</Cards>
```

Comment:
This needs to be external link - I think we have a component for that, let me know if you can't find it (check cards from the community links at the bottom)